src/project-2.js

```diff
@@ -95,77 +81,43 @@ const isPrime = (num) => {
   return true;
 };
 
-const returnFirst = (arr) => {
-  // return the first item from the array
-  return arr.shift();
-};
-
-const returnLast = (arr) => {
-  // return the last item of the array
-  return arr.pop();
-};
-
-const getArrayLength = (arr) => {
-  // return the length of the array
-  return arr.length;
-};
-
-const incrementByOne = (arr) => {
-  // arr is an array of integers
-  // increase each integer by one
-  // return the array
-  const incrementedArray = arr.map(element => element += 1);
-
-  return incrementedArray;
-};
-
-const addItemToArray = (arr, item) => {
-  // add the item to the end of the array
-  // return the array
-  arr.push(item);
+// return the first item from the array
+const returnFirst = arr => arr[0];
 
-  return arr;
-};
+// return the last item of the array
+const returnLast = arr => arr[arr.length - 1];
 
-const addItemToFront = (arr, item) => {
-  // add the item to the front of the array
-  // return the array
-  // hint: use the array method .unshift
-  arr.unshift(item);
+// return the length of the array
+const getArrayLength = arr => arr.length;
 
-  return arr;
-};
+const incrementByOne = arr => arr.map(element => element += 1);
```

Comment:
No need to update `element` here, as the result is implicitly returned from the arrow function anyway. `element + 1` will work and skip the assignment.